### PR TITLE
www-misc/buku: Bump to 3.7

### PR DESCRIPTION
**Pull request by jubalh**
Package-Manager: Portage-2.3.24, Repoman-2.3.6

### CODE REVIEW

File: www-misc/buku/buku-3.7.ebuild [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+	>=dev-python/urllib3-1.13.1[${PYTHON_USEDEP}]
+"
+
+DEPEND="${RDEPEND}
+	test? (
+		dev-python/flake8[${PYTHON_USEDEP}]
+		dev-python/hypothesis[${PYTHON_USEDEP}]
+		dev-python/py[${PYTHON_USEDEP}]
+		dev-python/pytest[${PYTHON_USEDEP}]
+		dev-python/pytest-catchlog[${PYTHON_USEDEP}]
```
Comment (stkw0) [inline www-misc/buku/buku-3.7.ebuild]: This dependency should be removed. Seems it have been merged and buku also removed it from their dependencies.

**Comment by jubalh** [comment]
Thanks for the feedback @stkw0. Sorry for the long waiting time to act upon it.

**Comment by a17r** [comment]
@stkw0 do you approve?

**Comment by stkw0** [comment]
@a17r Yes

File: www-misc/buku/buku-3.7.ebuild [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+inherit bash-completion-r1 distutils-r1
+
+DESCRIPTION="Powerful command-line bookmark manager"
+HOMEPAGE="https://github.com/jarun/Buku"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="amd64 ~x86"
```
Comment (a17r) [inline www-misc/buku/buku-3.7.ebuild]: You can't bump straight to stable.

**Pull request closed without merging**